packages/deployments/contracts/contracts/interfaces/IConnext.sol

```diff
@@ -91,17 +91,36 @@ interface IConnext {
    * @param amount - The amount of liquidity the router provided or the bridge forwarded, depending on
    * if fast liquidity was used
    * @param feePercentage - The amount over the BASEFEE to tip the relayer
+   * @param originSender - The sender of the transfer on the origin chain (i.e. the user, but not
+   * necessarily the xcaller).
    */
   struct ExecuteArgs {
     CallParams params;
     address local;
     address[] routers;
+    bytes[] signatures;
     uint32 feePercentage;
     uint256 amount;
     uint256 nonce;
     bytes relayerSignature;
     address originSender;
   }
+
+  /**
+   * @notice The permit represents the router's consent to use its funds to `execute` a transfer. This
+   * information should be hashed and signed by each router in the routers array individually.
+   * @param params - The CallParams. These are consistent across sending and receiving chains.
+   * @param local - The local asset for the transfer, will be swapped to the adopted asset if
+   * appropriate
+   * @param quorum - The length of the routers array. Indicates how many routers this router assents to
+   * conducting a multipath transfer with.
+   */
+  struct RouterPermit {
+    bytes transferId;
+    CallParams params;
+    address local;
+    uint16 quorum;
```

Comment:
we shouldn't call this a quorum -- its the number of routers in the path (they all have to support it, not just a subset). maybe just call it `pathLength`

Reply:
also since were not using this anywhere we should really delete it

Reply:
Yep, will do. Was initially using it lol